### PR TITLE
Update

### PR DESCRIPTION
**Pull request by Shaardeep**
I have Update the new nav list and add it in router also

### CODE REVIEW

File: Frontend/src/App.js [hunk 1]
```diff
@@ -1,20 +1,31 @@
...
+        <Header logo={logo} />
         <Routes>
           <Route path="/" element={<Home />} />
+          <Route path="/services" element={<Services />} />
+          <Route path="/Package" element={<Package />} />
+          <Route path="/about" element={<About />} />
+          <Route path="/contact" element={<Contact />} />
           <Route path="/register" element={<Signup />} />
           <Route path="/login" element={<Login />} />
+          <Route path="/service_detail?${id}" element={<ServiceDetail />} />
```
Comment (PatelNeelMahesh) [inline Frontend/src/App.js:28]: Add a '*' route as well.

File: Frontend/src/Pages/Services.jsx [hunk 2]
```diff
@@ -0,0 +1,13 @@
+import React from "react";
+import Service from "../comopnents/Service/Service";
+
+const Services = () => {
+  return (
+    <>
+      <Service />
+      <a href=""></a>
```
Comment (PatelNeelMahesh) [inline Frontend/src/Pages/Services.jsx:8]: Since you have routing it's better to use the Link tag from react-router-dom

Comment (Shaardeep) [reply Frontend/src/Pages/Services.jsx:8]: Ok sir

File: Frontend/src/App.js (hunk 1 above)
Comment (Shaardeep) [reply Frontend/src/App.js:28]: Ok sir

**Pull request merged**